### DISCUSSION TITLE
Why move build to proj instead of using CLI?

### DISCUSSION THREAD

**Discussion by halcwb** [TheAngryByrd/MiniScaffold#249] [Q&A] [answered]
Just a simple question. Why did you move the build to a proj instead of  @using the CLI?

**Comment by TheAngryByrd** [comment, marked as answer]
Can you define what you mean by CLI?

**Reply by halcwb** [reply]
I am sorry I mean using the fake command line interface. So it used to be a command like ``fake build -t tests``, that started a build.fsx. I know there is some movement in the Fake maintenance situation.

**Reply by TheAngryByrd** [reply]
I listed the reasons in the PR for it https://github.com/TheAngryByrd/MiniScaffold/pull/246

Basically it comes down to Ionide playing nice with the file and fake-cli running into issues as dotnet keeps moving forward. 

If fake-cli is getting some love I’ll be happy to move back.

**Reply by halcwb** [reply]
Crystal clear. Thanks.
